lerna.json

```diff
@@ -2,5 +2,10 @@
   "packages": [
     "packages/*"
   ],
-  "version": "independent"
+  "version": "independent",
+  "command": {
+    "version": {
+      "includeMergedTags": true
```

Comment:
More on this addition is included in the comment for the commit that introduced it: https://github.com/apollographql/apollo-tooling/pull/1501/commits/873349fdcbfb654329f159af91013425fe7f6cce